about.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html> 
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title> About Alaere </title>
+    <link rel="stylesheet" href="styles/style.css">
+</head>
+<body>
+  <nav>
+      <ul class="nav-list">
+        <li><a href="http://127.0.0.1:5500/index.html"> Home </a></li>
+        <li><a href="http://127.0.0.1:5500/about.html"> About </a></li>
```

Comment:
Consider making the nav link for the current page inactive (i.e., remove the `a` tag). Doing so can help communicate to a visitor that they are already on this page.